src/events/maxisonline.py

```diff
@@ -114,8 +114,9 @@ async def on_presence_update(self, before: discord.Member, after: discord.Member
             # Get Max's info only
             if before.id == self.max_member.id:
                 if before.guild.id == 730196305124655176:
-                    if (before.status != discord.Status.online and after.status == discord.Status.online) \
+                    if (before.is_on_mobile() and after.status == discord.Status.online) \
```

Comment:
Why not `before.is_on_mobile() != after.is_on_mobile()`? It's possible to be mobile and online so this doesn't catch all cases.

Reply:
ok